src/applications/disability-benefits/526EZ/tests/config/primaryAddress.unit.spec.jsx

```diff
@@ -78,6 +79,50 @@ describe('Disability benefits 526EZ primary address', () => {
     expect(form.find('input').length).to.equal(7);
   });
 
+  it('restricts state options to military state codes when city is a military city code', () => {
```

Comment:
Should we test the different flows you mentioned in your comment above (immediate validation vs. validation on submit) if we aren't already?